aspnetcore/grpc/security.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Security considerations in gRPC for ASP.NET Core
+author: jamesnk
+description: Learn about security considerations for gRPC for ASP.NET Core.
+monikerRange: '>= aspnetcore-3.0'
+ms.author: jamesnk
+ms.custom: mvc
```

Comment:
what is this?

Reply:
It is in all the asp.net core docs. I don't know what it is for.